app/src/main/java/com/example/wizelineandroid/core/RetrofitClient.kt

```diff
@@ -0,0 +1,36 @@
+package com.example.wizelineandroid.core
+
+import com.example.wizelineandroid.repository.WebService
+import com.example.wizelineandroid.utils.AppConstants
+import com.google.gson.GsonBuilder
+import okhttp3.OkHttpClient
+import okhttp3.logging.HttpLoggingInterceptor
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+//peticion a datasource, covertir json de api a movie
+object RetrofitClient{
+
+    private val interceptor: HttpLoggingInterceptor = HttpLoggingInterceptor().apply {
+        this.level = HttpLoggingInterceptor.Level.BASIC
+    }
+
+    private const val userAgent = ""
+
+    private val client =
+        OkHttpClient.Builder().addInterceptor(interceptor).addNetworkInterceptor { chain ->
```

Comment:
It would be nicer to create this interceptor in its own variable, like the logging interceptor.